tests/__init__.py

```diff
@@ -0,0 +1,147 @@
+    input_subjects.subject_description = ['dl56', 'SC035', 'SC038', 'oe_talab', 'rich']
+
+    subjects_csv_fp = pathlib.Path('./tests/user_data/subjects.csv')
+
+    input_subjects.to_csv(subjects_csv_fp)  # write csv file
+
+    yield input_subjects
+
+    subjects_csv_fp.unlink()  # delete csv file after use
+
+
+@pytest.fixture
+def sessions_csv():
+    """ Create a 'sessions.csv' file"""
+    input_sessions = pd.DataFrame(columns = ['subject', 'session_dir'])
+    input_sessions.subject = ['subject1',
+                              'subject2',
+                              'subject2',
+                              'subject3',
+                              'subject4',
+                              'subject5']
+    input_sessions.session_dir = [r'F:\U24\workflow_ephys_data\subject1\session1',
+                                  r'F:\U24\workflow_ephys_data\subject2\session1',
+                                  r'F:\U24\workflow_ephys_data\subject2\session2',
+                                  r'F:\U24\workflow_ephys_data\subject3\session1',
+                                  r'F:\U24\workflow_ephys_data\subject4\experiment1',
+                                  r'F:\U24\workflow_ephys_data\subject5\2018-07-03_19-10-39']
+
+    sessions_csv_fp = pathlib.Path('./tests/user_data/sessions.csv')
+
+    input_sessions.to_csv(sessions_csv_fp)  # write csv file
+
+    yield input_sessions
+
+    sessions_csv_fp.unlink()  # delete csv file after use
+
+
+@pytest.fixture
+def testdata_paths():
+    return {
+        'npx3A-p0-ks': 'subject5/2018-07-03_19-10-39/probe_1/ks2.1_01',
+        'npx3A-p1-ks': 'subject5/2018-07-03_19-10-39/probe_1/ks2.1_01',
+        'oe_npx3B-ks': 'subject4/experiment1/recording1/continuous/Neuropix-PXI-100.0/ks',
+        'npx3A-p0': 'subject5/2018-07-03_19-10-39/probe_1',
+        'oe_npx3B': 'subject4/experiment1/recording1/continuous/Neuropix-PXI-100.0',
+    }
+
+
+@pytest.fixture
+def kilosort_paramset():
+    from workflow_ephys.pipeline import ephys
+
+    params_ks = {
+        "fs": 30000,
+        "fshigh": 150,
+        "minfr_goodchannels": 0.1,
+        "Th": [10, 4],
+        "lam": 10,
+        "AUCsplit": 0.9,
+        "minFR": 0.02,
+        "momentum": [20, 400],
+        "sigmaMask": 30,
+        "ThPr": 8,
+        "spkTh": -6,
+        "reorder": 1,
+        "nskip": 25,
+        "GPU": 1,
+        "Nfilt": 1024,
+        "nfilt_factor": 4,
+        "ntbuff": 64,
+        "whiteningRange": 32,
+        "nSkipCov": 25,
+        "scaleproc": 200,
+        "nPCs": 3,
+        "useRAM": 0
+    }
+
+    # doing the insert here as well, since most of the test will require this paramset inserted
+    ephys.ClusteringParamSet.insert_new_params(
+        'kilosort2', 0, 'Spike sorting using Kilosort2', params_ks)
+
+    yield params_ks
+
+    with dj.config(safemode=False):
+        ephys.ClusteringParamSet.delete()
```

Comment:
You can reduce this further if you like. In `0.13.X` you can now pass a kwarg as `table.delete(safemode=False)` for one-off use. However, since setting `safemode=False` will likely recur, I would just set it in your first `dj_config` fixture.

Reply:
ah yes, good point, let me set that in the `dj_config` fixture

Reply:
Very cool! For the directory, is it possible to configure it to make the tests runnable also on other systems. 
`F:\U24\workflow_ephys_data\subject1\session1` only works for windows